x/evm/types/key.go

```diff
@@ -4,12 +4,12 @@ const (
 	// ModuleName string name of module
 	ModuleName = "evm"
 
-	// EvmStoreKey key for ethereum storage data
-	EvmStoreKey = "evmstore"
-	// EvmCodeKey key for ethereum code data
-	EvmCodeKey = "evmcode"
-	// EvmBlockKey key for ethereum block data
-	EvmBlockKey = "evmblock"
+	// StoreKey key for ethereum storage data (StateDB)
+	StoreKey = ModuleName
```

Comment:
Note that the StoreKey is juse used as a replacement for the storage trie in Ethereum. Seems slightly weird to me that it's used as the primary store when comparing with Ethereum, but I guess of these keys I suppose it would be